src/reaper_osara.cpp

```diff
@@ -4210,6 +4210,22 @@ void cmdSelectFromCursorToStartOfProject(Command* cmd) {
 	outputMessage(translate("selected to start of project"));
 }
 
+void cmdSetPhaseNormalAllTracks(Command* cmd) {
```

Comment:
cmd -> command for consistency. I messed this up in the last action I added, so this is my fault, not yours. :)

```suggestion
void cmdSetPhaseNormalAllTracks(Command* command) {
```